ui/components/datasetItemWorkspace/src/components/Features/AutoCompleteFeatureInput.svelte

```diff
@@ -24,6 +24,7 @@
   export let value: string = "";
   export let autofocus: boolean = false;
   export let className = "";
+  export let isInputEnabled: boolean = true;
```

Comment:
I'm wondering if  `isSearchInputEnabled` or just `isSearchEnabled` would be more relevant here?